src/pymodaq_utils/serialize/serializer.py

```diff
@@ -254,6 +254,104 @@ def deserialize(bytes_str: bytes) -> Tuple[List[SERIALIZABLE], bytes]:
         return list_obj, remaining_bytes
 
 
+class TupleSerializeDeserialize(SerializableBase):
```

Comment:
Instead of tuple serialization, we could use the list conversion in the ThreadCommand serialization.